docxcompose/composer.py

```diff
@@ -72,6 +72,7 @@ def insert(self, index, doc, remove_property_fields=True):
             self.add_numberings(doc, element)
             self.restart_first_numbering(doc, element)
             self.add_images(doc, element)
+            self.add_diagrams(doc, element)            
```

Comment:
Can you please remove the trailing spaces?

```suggestion
            self.add_diagrams(doc, element)
```